docs/gke/configs/cluster-kubeflow.yaml

```diff
@@ -60,16 +60,16 @@ resources:
           enabled: false
     # Path for the bootstrapper image.
     bootstrapperImage: gcr.io/kubeflow-images-public/bootstrapper:latest
-    # This is the name of the GCP static ip address to reserve for your domain.
-    # This must be different for each Kubeflow deployment in your project.
+    # This is the name of the GCP static ip address reserved for your domain.
+    # Each Kubeflow deployment in your project should use one unique ipName among all configs.
     ipName: kubeflow-ip
     # Provide the config for the bootstrapper. This should be a string
     # containing the YAML spec for the bootstrapper.
     #
     # You should set the following parameters in bootstrapperConfig.
     #    acmeEmail - Set this to your email address; this is used with lets-encrypt.
     #    ipName - This is the name of the static ip address reserved in GCP to be used with ingress.
-    #             keep this in sync with the value of property ipName
+    #             For each deployment, value of ipName should be same among all configs
```

Comment:
This comment is confusing. It makes it sound like every deployment should use the same value for ipName which isn't correct because we don't want different Kubeflow deployments to use the same name.

I would say something like

Keep the value of ipName in sync with the value in the bootstrapper config section below.